### DISCUSSION TITLE
Password Imports + bonus idea

### DISCUSSION THREAD

**Discussion by griffindavidson** [minbrowser/min#1991] [Ideas]
Lovely browser! Could you guys add support for importing passwords from plaintext? That would be fantastic and highly appreciated.

maybe as a bonus / stretch, as a college student stuck using Duo Authentication, is there any way to enable TouchID support for Duo with the browser? Duo supports Mac TouchID from Chrome, and the option appears here in min browser, but doesn't communicate with the touchID sensor. Just a stretch idea, I would love to see it come to min!

**Comment by PalmerAL** [comment]
I'm also stuck with Duo, it is fairly annoying :) I think we need this change in Electron to have touchID support, which looks like it won't be completed any time soon: https://github.com/electron/electron/pull/31906

The first one does sound like a good idea.
